waku/v2/node/wakunode2_test.go

```diff
@@ -32,3 +32,108 @@ func TestWakuNode2(t *testing.T) {
 
 	require.NoError(t, err)
 }
+
+// func Test1100(t *testing.T) {
```

Comment:
I had to comment out this test. The test is actually for the issue I identified last month, but seems to be facing a separate issue that is going to need investigation. I just didn't want to block this PR based on it.

The test is flaky. Sometimes it gets through all 1100 messages in 2-3 seconds, sometimes it gets stuck at 65 or 900 after 20s. All messages publish without error, but not all messages are read. I don't quite know why, but it's obviously something serious we are going to want to get to the bottom of. Also seems to work more reliably on my super fast laptop than in CI, which is maybe a hint.

I can bump the number of messages to 2k, and it will get stuck at a number > 1057, which is why I think it's a separate issue from the one I identified (which always got stuck at the exact same point).

Reply:
Might be worth to paste this explanation into the comment, so that we don't have to hunt for it when we get back to it.